lib/i18n_rails_helpers/model_helpers.rb

```diff
@@ -0,0 +1,45 @@
+module I18nRailsHelpers
+  module ModelHelpers
+    # enum attrubute_t and attributes_t return translated enum values
+    #
+    # Example:
+    #   in the Client model
+    #     enum gender: { undefined: 0, female: 1, male: 2 }
+    #   in use
+    #     Client.first.gender    # => 'female'
+    #     Client.first.gender_t  # => 'Frau'
+    #     Client.first.genders_t # => { undefined: 'Nicht definiert', female: 'Frau', male: 'Mann' }
+    #
+    # Requires:
+    #   locale key: activerecord.attributes.#{model_name}.#{enum}.#{enum_value_key}
+    #   eg.:        activerecord.attributes.client.gender.female                      # => 'Frau'
+    #
+    def define_enum_t_methods
+      defined_enums.each do |enum_attr, values|
+        self.class.send(:define_method, "#{enum_attr}_t") do
+          public_send(:t_enum, enum_attr)
```

Comment:
You can also just do `t_enum(enum_attr)` here ;-) `send` is only needed if the method name is dynamic.